src/main/java/net/pms/dlna/DLNAMediaInfo.java

```diff
@@ -877,6 +877,7 @@ public void parse(InputFile inputFile, Format ext, int type, boolean thumbOnly,
 					// Create the thumbnail image using the Thumbnailator library
 					try {
 						thumb = ImagesUtil.scaleImage(Files.readAllBytes(file.toPath()), 320, 320, ScaleType.MAX, ImageFormat.JPEG, false);
+						ffmpeg_parsing = false;
```

Comment:
@Nadahar becuause the condition `if (!thumbOnly || !configuration.isUseMplayerForVideoThumbs()) {` is true for `!configuration.isUseMplayerForVideoThumbs()` in that case.